web/src/main/webapp/dataViews/schemaView.ts

```diff
@@ -184,6 +187,10 @@ export class SchemaView extends TSViewBase {
             descriptions.push("Minimum value", "Maximum value", "Average value",
                 "Standard deviation", "Number of missing elements");
         }
+        if (this.counts != null) {
+            names.push("DistinctCount");
```

Comment:
Let's call it EstDistinct... Or use an approx sign

Reply:
Added an approximation sign